### DISCUSSION TITLE
More colors?

### DISCUSSION THREAD

**Discussion by poperigby** [dracula/dracula-theme#597] [Q&A] [answered]
Hey everyone! I really love Dracula theme, but some times I find my self with not enough colors to do what I want. What do other people do when they have this problem? Are there maybe some other colors that can extend the theme a bit?

**Comment by benknoble** [comment]
I occasionally find myself with the same problem--my biggest difficulty is the minimal contrast between a color and its brightened variant.

The best thing would be for Zeno to come up with some more accessibly-contrasted alternatives. Additionally, it might be nice to have a blue that isn't purple or cyan and a brown that isn't yellow or orange. I'm no good for these kinds of design things, but this is what I would like to see @zenorocha

**Reply by poperigby** [reply]
Agreed.

**Reply by luxonauta** [reply, marked as answer]
Planed for 2024! 💜

**Reply by tangowithfoxtrot** [reply]
@luxonauta, were more colors ever added? I find myself having to improvise various colors that are in between the 12 colors that are officially designated when creating themes for other applications.

**Reply by luxonauta** [reply]
Hey! 👋🏻 I'm coming back to this thread to say that in the [new port,](https://github.com/dracula/cursor/blob/main/dev/src/alucard.yml) there are already four more colours to be used in the UI, and more will be added to the spec this week. 🧩
